timesketch/lib/testlib.py

```diff
@@ -534,6 +556,25 @@ def _create_searchtemplate(self, name, user):
         self._commit_to_database(searchtemplate)
         return searchtemplate
 
+    def _create_sigma(
```

Comment:
In the interest of keeping the test logic as straightforward as possible, I would omit this thin wrapper and just call `Sigma()` in `setUp`.

Reply:
hm I just followed the example of other create methods in that same file